roles/libvirt_manager/tasks/create_networks.yml

```diff
@@ -81,6 +82,112 @@
   loop_control:
     label: "{{ item.name }}"
 
+## Ensure we get dnsmasq DHCP service for all created networks
+# Refreshing network facts will ensure we get the new interfaces.
+# They (usually?) have the same name as the network itself.
+- name: Refresh networking facts on host
+  ansible.builtin.setup:
+    gather_subset:
+      - '!all,!min'
+      - network
+
+- name: Build needed network related content
+  when:
+    - ansible_facts[_name] is defined
+  vars:
+    _name: "{{ item | replace('-', '_') }}"
+    _no_prefix_name: "{{ _name | regex_replace('cifmw[-_]','') }}"
+    _fixed_nets: >-
+      {{
+        cifmw_libvirt_manager_fixed_networks_defaults +
+        cifmw_libvirt_manager_fixed_networks
+      }}
+    _dns_options: |-
+      {% if ansible_facts[_name].ipv4 is defined -%}
+      - "option:dns-server,{{ ansible_facts[_name].ipv4.address }}"
+      {% endif -%}
+      {% if ansible_facts[_name].ipv6 is defined -%}
+      - "option6:dns-server,[{{ ansible_facts[_name].ipv6.address }}]"
+      {% endif -%}
+    _dns_listener:
+      - "{{ ansible_facts[_name].ipv4.address | default('') }}"
+      - "{{ ansible_facts[_name].ipv6.address | default('') }}"
+    _dns_addr: |-
+      {% if _no_prefix_name is match('^[a-z0-9]bm$')%}
```

Comment:
Why are we adding these in `create_networks.yml`? These are DNS entries for OpenShift, I don't think it belongs here. We only need one entry for each of these right? Can we not make it simple and just require this in the input?

Reply:
I'll see if I can move them out, for instance in the reproducer role instead.